internal/manifests/collector/container.go

```
@@ -165,7 +165,7 @@ func getConfigContainerPorts(logger logr.Logger, cfg string) map[string]corev1.C
 		logger.Error(err, "couldn't extract the configuration")
 		return ports
 	}
-	ps := adapters.ConfigToPorts(logger, c)
+	ps, _ := adapters.ConfigToPorts(logger, c)
```

suggestion: we should also propagate the error up here if we can